src/dialects/mssql/index.js

```diff
@@ -138,6 +140,18 @@ assign(Client_MSSQL.prototype, {
     })
   },
 
+  // sets a request input parameter. Detects bigints and sets type appropriately.
+  _setReqInput(req, i, binding) {
+    if (typeof binding == 'number' && (binding < -2147483648 || binding > 2147483647)) {
+      if (binding < -9007199254740991 || binding > 9007199254740991) {
```

Comment:
Ok for the constants, isSafeInteger wasn't transpiled or polyfiled (I guess this is a matter of babel profile and plugin) Maybe constants would be the most compatible approach?